<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>190</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Pay</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rsc.qrc">
    <normaloff>:/icons/icons/tree.png</normaloff>:/icons/icons/tree.png</iconset>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>221</width>
     <height>281</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(248, 220, 209);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="rsc.qrc">:/images/images/sky.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="ID_Number_LineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 2px solidrgba(0, 0, 0, 150);
	border-radius: 10px;
	color: rgb(0, 0, 0);
	padding-left: 20px;
	padding-right: 2px;
	background-color: rgb(255, 255, 255);
}
</string>
   </property>
   <property name="placeholderText">
    <string>ID NUMBER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RegAmount_PushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0.397727 rgba(68, 164, 250, 255), stop:1 rgba(255, 255, 255, 255));
	
	color: rgba(255, 255, 255, 210);
	border-radius: 12px;
}


QPushButton:pressed {
	padding-left: 5px;
	padding-top: 5px;
	
}</string>
   </property>
   <property name="text">
    <string>Register Amount</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Amount_LineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
	border: 2px solidrgba(0, 0, 0, 150);
	border-radius: 10px;
	color: rgb(0, 0, 0);
	padding-left: 20px;
	padding-right: 2px;
	background-color: rgb(255, 255, 255);
}
</string>
   </property>
   <property name="placeholderText">
    <string>AMOUNT</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
